src/org/labkey/test/tests/announcements/ModeratorReviewTest.java

```diff
@@ -159,12 +159,15 @@ private String insertMessage(String user, String title, boolean expectAutoApprov
         goToProjectHome();
         impersonate(user);
         log("Insert message");
+        log("Currently impersonating user:" + user);
         // make title unique in test scope, to check email notifications later
         title = title + " " + System.currentTimeMillis();
+        log("Inserting message with title " + title);
         InsertPage.beginAt(this)
                 .setTitle(title)
                 .setBody(title)
                 .submit();
+        log("Submission of message (supposedly) complete");
```

Comment:
This was a for-my-own debugging message, maybe it should be removed?

Reply:
Extra logging in a test case is not a bad thing, and it may turn out to be useful at some future date.